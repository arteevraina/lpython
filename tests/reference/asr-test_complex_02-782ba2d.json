{
    "basename": "asr-test_complex_02-782ba2d",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/test_complex_02.py",
    "infile_hash": "e6ee66d952deccec11d316a09f1d5b4bb4916ceb56a41a11dee03ae9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-test_complex_02-782ba2d.stdout",
<<<<<<< HEAD
    "stdout_hash": "25b57ce3115249491b17231131595fe5b6e48450a5a0ae64d953c1c6",
=======
    "stdout_hash": "065f842790f986f276b50c69fb831c21de22dcb13fb0e410256ece9d",
>>>>>>> main
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}