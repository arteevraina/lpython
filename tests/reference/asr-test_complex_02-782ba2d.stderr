warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
 --> tests/../integration_tests/test_complex_02.py:5:13
  |
5 |     x = c32(complex(3, 4))
  |             ^^^^^^^^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
